docs/source/commandline/commandline-all.rst

```diff
@@ -63,20 +63,18 @@ executable (for example you could call it ``mame-here`` )::
 
     #!/bin/sh
     cd "`dirname "$0"`"
-    exec ./mame64
+    exec ./mame
 
 You should be able to use any text editor.  If you have a choice of file format
-or line ending style, choose UNIX.  This assumes you're using a 64-bit release
-build of MAME, but if you aren't you just need to change ``mame64`` to the name
```

Comment:
You can still end up with `mamed`, `mamep`, `mamedp` etc. depending on other options – should the note about possibly needing to change it be left in and just adjusted?

Reply:
Adjusted this, let me know if that meets your expectations.